src/main/java/org/opentripplanner/routing/graph/SerializedGraphObject.java

```diff
@@ -112,6 +113,8 @@ public static Kryo makeKryo() {
         // We might actually want to manually register a serializer for every class, to be safe.
         kryo.setRegistrationRequired(false);
         kryo.setReferences(true);
+        // We serialize in two steps, graph and edges. Reference tracking must not be reset in between.
```

Comment:
What does this fix? It seems to be a separate change not related to this pull request.

Reply:
Yes, this is not needed. The Serailizer for HashMultimap however is needed for the childStops in Station to be correctly serailized. Otherwise it fails with the following error when starting up.

```
13:09:15.015 ERROR (OTPMain.java:43) An uncaught error occurred inside OTP: Error during Java deserialization.
Serialization trace:
transfersByStop (org.opentripplanner.routing.graph.Graph)
graph (org.opentripplanner.routing.graph.SerializedGraphObject)
com.esotericsoftware.kryo.KryoException: Error during Java deserialization.
Serialization trace:
transfersByStop (org.opentripplanner.routing.graph.Graph)
graph (org.opentripplanner.routing.graph.SerializedGraphObject)
	at com.esotericsoftware.kryo.serializers.JavaSerializer.read(JavaSerializer.java:68) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:731) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:125) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:543) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:731) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:125) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at com.esotericsoftware.kryo.serializers.FieldSerializer.read(FieldSerializer.java:543) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at com.esotericsoftware.kryo.Kryo.readClassAndObject(Kryo.java:813) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at org.opentripplanner.routing.graph.Graph.load(Graph.java:711) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at org.opentripplanner.routing.graph.Graph.load(Graph.java:702) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at org.opentripplanner.standalone.OTPMain.startOTPServer(OTPMain.java:100) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at org.opentripplanner.standalone.OTPMain.main(OTPMain.java:36) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
Caused by: java.lang.NullPointerException: null
	at org.opentripplanner.model.TransitEntity.hashCode(TransitEntity.java:43) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at java.base/java.util.HashMap.hash(HashMap.java:339) ~[na:na]
	at java.base/java.util.HashMap.put(HashMap.java:612) ~[na:na]
	at java.base/java.util.HashSet.readObject(HashSet.java:342) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:567) ~[na:na]
	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1160) ~[na:na]
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2271) ~[na:na]
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2142) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1646) ~[na:na]
	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2410) ~[na:na]
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2304) ~[na:na]
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2142) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1646) ~[na:na]
	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2410) ~[na:na]
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2304) ~[na:na]
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2142) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1646) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:464) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:422) ~[na:na]
	at com.google.common.collect.Serialization.populateMultimap(Serialization.java:173) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at com.google.common.collect.HashMultimap.readObject(HashMultimap.java:136) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:567) ~[na:na]
	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1160) ~[na:na]
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2271) ~[na:na]
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2142) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1646) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:464) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:422) ~[na:na]
	at com.esotericsoftware.kryo.serializers.JavaSerializer.read(JavaSerializer.java:66) ~[otp-2.0.0-SNAPSHOT-shaded.jar:1.1]
	... 11 common frames omitted

```